shell/platform/android/io/flutter/view/ResourceExtractor.java

```diff
@@ -202,17 +203,23 @@ private String checkTimestamp(File dataDir) {
         final String[] existingTimestamps = getExistingTimestamps(dataDir);
 
         if (existingTimestamps == null) {
-            Log.i(TAG, "No extracted resources found");
+            if (BuildConfig.DEBUG) {
+                Log.i(TAG, "No extracted resources found");
+            }
             return expectedTimestamp;
         }
 
         if (existingTimestamps.length == 1) {
-            Log.i(TAG, "Found extracted resources " + existingTimestamps[0]);
+            if (BuildConfig.DEBUG) {
+                Log.i(TAG, "Found extracted resources " + existingTimestamps[0]);
+            }
         }
 
         if (existingTimestamps.length != 1
                 || !expectedTimestamp.equals(existingTimestamps[0])) {
-            Log.i(TAG, "Resource version mismatch " + expectedTimestamp);
+            if (BuildConfig.DEBUG) {
+                Log.i(TAG, "Resource version mismatch " + expectedTimestamp);
```

Comment:
Same for this one - perhaps a `Log.w` or `Log.e`?

Reply:
Same here.